software/sys_controller/ossc/avconfig.h

```diff
@@ -48,6 +48,22 @@
 #define L5FMT_1600x1200     1
 #define L5FMT_1920x1200     2
 
+static const char *avinput_str[] = { "Last used", "AV1: RGBS", "AV1: RGsB", "AV1: YPbPr", "AV2: YPbPr", "AV2: RGsB", "AV3: RGBHV", "AV3: RGBS", "AV3: RGsB", "AV3: YPbPr", "Test pattern" };
+
+typedef enum {
+    AV_LAST         = 0,
```

Comment:
AV_LAST is used in array size specifier, changing the order currently breaks some functionality.

Reply:
Doh! I noticed that in @borti4938 's original branch, but for some reason failed to fix it as part of 
a5e25d2 in my own version. Sorry about that! Will fix.

Reply:
Well, I see that it was a bit spreaded over commits; so it might be confusing. I also forgot to add correct initialization of input_profiles, which should be input_profiles[AV_TESTPAT] with this order.

Reply:
The more I look into this, the more convinced I am that changing avinput_t was not a good idea. There are arrays which used AV_LAST to denote their size. However, even if their sizes are changed to AV_TESTPAT, the indices now cover the range [AV_LAST,  AV3_YPBPR] instead of the former [AV_TESTPAT, AV3_YPBPR]. This is bad and would require indexing fixups in many places.

Reply:
There are just a few changes which have to be made. Everything related to input_profiles[].